api/tests-static/test_static_api_handlers.cc

```diff
@@ -34,7 +34,19 @@ TEST_F(TestStaticClientHandler, AwaStaticClient_SetResourceOperationHandler_Inva
 {
     auto client = AwaStaticClient_New();
     ASSERT_TRUE(client != NULL);
-    EXPECT_EQ(AwaError_Success, AwaStaticClient_DefineObject(client, 9999, "TestObject", 0, 1));
+
+    EXPECT_EQ(AwaError_DefinitionInvalid,             AwaStaticClient_SetResourceOperationHandler(client, 9999, 1, handler));
```

Comment:
Perhaps it would be better to create a new error code, suggest _AwaError_StaticClientNotInitialized_? Then users know that their calls are failing because of that reason.